### ISSUE TITLE
Newly crafted item has durability of an old one

### ISSUE THREAD

**Issue by edwardexo** [overextended/ox_inventory#1056]
**Describe the bug**
Sometimes when you craft an item which you have previously had in your inventory, the newly crafted item will have the durability of the item you once had in your inventory.

**To Reproduce**
1. Have an item with a durability of 40 in your inventory
2. Drop / stash the item somewhere 
3. Open a crafting bench and craft the same item

**Expected behavior**
The newly crafted item has 100 durability (as no metadata is defined in the crafting bench code).

**Actual behaviour**
The newly crafted item has 40 durability (same as the item you once held).

**Screenshots**
Clip 1 of it happening (50 durability repair kit): https://streamable.com/jod3lt
Clip 2 of it happening (55 durability sponge): https://streamable.com/jwhnb6

**Additional context**
Unfortunately I was unable to replicate the error on my test server. Opening this report in hopes you might have some insight why is this happening. The two clips provided are from two different players in the live server. In the crafting bench code I don't define any metadata, so they should be crafted with 100 durability instead of 50/55 like they did.

```
{
    name = 'sponge',
    ingredients = {
        water = 1,
	plastic = 1
    },
    duration = 3000,
    count = 1,
    metadata = {}
},
```

**Comment by thelindat** [comment]
Haven't been able to reproduce with the given steps.

**Comment by edwardexo** [comment]
@thelindat I can now reliably reproduce this bug. New steps to reproduce:

1. Craft a new usable item (100 durability)
2. Use the item with `consume = 0.1` (so it will have 90 durability after the use)
3. Go and craft the item again

The crafted item will now have 90 durability (when it should be 100).

**It seems that using an item is changing the crafting recipe metadata.**

When I added debug messages, this is what I observed:

* When the crafting bench is created on inventory start, the metadata is empty (`metadata = {}`)
* After you craft the item for the first time, the crafting bench recipe metadata is now` durability = 100`
* When you use the item, the crafting bench recipe changes to `durability = 90.0.`

The crafting bench recipe metadata is permanently changed after you have used the item. Now you, and all the other players will craft the item with 90 durability. Also a video, done straight after an inventory restart with the steps I posted: https://streamable.com/dhgceh
